client/blocks/site-address-changer/index.jsx

```diff
@@ -384,14 +392,18 @@ export class SiteAddressChanger extends Component {
 				<div className="site-address-changer__info">
 					<p>
 						{ translate(
-							'Once you change your site address, %(currentDomainName)s will no longer be available. {{a}}Did you want to add a custom domain instead?{{/a}}',
+							'Once you change your site address, %(currentDomainName)s will no longer be available. ',
```

Comment:
Could you do two different strings here so we maintain the old translation and avoid breaking the sentences up? (shouldn't concat text as a general i18n rule)

Reply:
Updated!